scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -65,6 +65,23 @@ enum ContainerOs {
     Windows
 }
 
+function New-Sockets([string] $EdgeDataDirectory) {
```

Comment:
AFAICT it would've been sufficient to move this code from `Deploy-` code path to `Initialize-`. I don't see why you need to extract it into a function and then call it from two different places.

Reply:
since everything else was already moved from deploy to initialize i assumed it was being done in deploy for some scenario i didn't understand.  if you want me to just move it then i can do that.

Reply:
No, there's no reason it needs to be in `Deploy-`

Reply:
ok, i left it in the new function but am only calling it from init.

Reply:
You're calling it in `Get-IoTEdge` too which gets called from `Deploy-`.